cmd/process-agent/collector.go

```diff
@@ -125,6 +128,11 @@ func NewCollectorWithChecks(cfg *config.AgentConfig, checks []checks.Check, runR
 	podResults := api.NewWeightedQueue(queueSize, int64(cfg.Orchestrator.PodQueueBytes))
 	log.Debugf("Creating pod check queue with max_size=%d and max_weight=%d", podResults.MaxSize(), podResults.MaxWeight())
 
+	dropConnectionsCheck := ddconfig.Datadog.GetBool("process_config.drop_connections_check")
```

Comment:
I could see some uses in generalizing it beyond just the `connections` check for one area future work. What do you think about `process_config.drop_check_payloads` or similar that would be a list of check names that would not report their results. cc @mbotarro

Reply:
Having a list of check names to be dropped seems a good approach to follow. We can probably check if `result.name` is in the list of blocked checks and drop the payload before hitting the switch statement in the `consumePayloads` function.